packages/gasket-plugin-nextjs/README.md

```diff
@@ -207,6 +207,24 @@ module.exports = {
 };
 ```
 
+### nextPreHandling
+
+Enables execution of custom logic just prior to an HTTP request being handed to next.js for processing. Hooks receive the request, response, and next server (not to be confused with the next function used by express-like middleware):
+
+```js
+module.exports = {
+    hooks: {
+        async nextPreHandling(gasket, {
+            req,
+            res,
+            next
```

Comment:
I think this needs a different name, (perhaps `nextServer`) since `next` is the common name for middleware callbacks typically seen alongside req and res ([express docs](http://expressjs.com/en/guide/using-middleware.html#middleware.application))

Reply:
Hmm, I see what you're saying, however, I feel like the `preHandling` verbiage is helpful in describing the lifecycle. Hmm. I wonder if we should name this lifecycle based on when it is expected to run? Or should we name it based on what the most common usage is for it, like `preRequestHandling`?

Reply:
I think this was about naming the `next` property passed to the lifecycle `nextServer`. If we did want to rename the _lifecycle_, please make sure to include `next` in the name; `preRequestHandling` is ambiguous, but `nextPreRequestHandling` or `preNextRequestHandling` would be more descriptive.